src/structures/ClientUser.ts

```diff
@@ -0,0 +1,44 @@
+import { Emote } from "./Emote";
+import { EmoteSet } from "./EmoteSet";
+import { Role } from "./Role";
+import { User } from "./User";
+
+export class ClientUser implements User {
```

Comment:
Why is this a class?

Reply:
I was thinking of abstracting some stuff but it's actually not needed now, since there is the actor store. I'll get rid of that